docker-compose.yml

```diff
@@ -204,7 +205,19 @@ services:
     environment:
       MINIO_ACCESS_KEY: ${MINIO_ACCESS_KEY:-usaspending}
       MINIO_SECRET_KEY: ${MINIO_SECRET_KEY:-usaspender}
-    command: server --address ":10001" --console-address ":10002" /data
+    entrypoint: >
+      /bin/sh -c "
+        if [ ! -d data/dti-da-public-files-nonprod ]; then
```

Comment:
These blocks can be simplified further. There is no need to check if the directory already exists because mkdir will create it only if it doesn't already exist.

> The mkdir command will create any folders that do not exist in the specified path

Reply:
You can remove the if thens, as long as an error doesn't occur and interrupt execution.